src/AppBundle/Converter/TwigTemplateConverter.php

```diff
@@ -206,6 +209,13 @@ private function convertLayout($mjmlTheme, $newTheme)
         //Update assets path
         $innerHtml = preg_replace('#'.$mjmlTheme.'/assets/#', $newTheme.'/assets/', $innerHtml);
 
+        //if mj-section is inside mj-wrapper, we need to remove the conditional `if mso table`
+        if ($isWrapped) {
+            $innerHtml = preg_replace('#^<!--\[if mso \| IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0">#', '<!--[if mso | IE]>', $innerHtml);
```

Comment:
preg_replace or str_replace? I don't see a regexp here.

Reply:
Oh, I enclosed my regexes with `#` in order not to escape `/` (that do not occur here anyway...). I recognize this is not standard, so I change it.